cgmanifest.json

```diff
@@ -55,8 +55,8 @@
         "type": "other",
```

Comment:
Do we need a cgmanifest entry for the downloaded openjdk? Also, did we ever track the removed OpenJDK raw toolchain dependencies here?

Reply:
Thanks for the reminder. For openjdk raw dependencies these were added in #1096 . Will remove these

Reply:
And yes I think that the downloaded openjdk needs to be in cgmanifest as well. This has been added.